bitnami/argo-cd/templates/cluster-configs.yaml

```diff
@@ -0,0 +1,25 @@
+{{- range .Values.config.clusterCredentials }}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "argocd.fullname" $ }}-cluster-{{ .name }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+    # Mandatory label
+    # Ref: https://argoproj.github.io/argo-cd/operator-manual/declarative-setup/#atomic-configuration
+    app.kubernetes.io/part-of: argocd
+    argocd.argoproj.io/secret-type: cluster
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
+type: Opaque
+stringData:
+  name: {{ required "A valid .Values.config.clusterCredentials[].name entry is required!" .name }}
```

Comment:
This would normally go in a validateValues function, so we can show all the errors